### PR TITLE
gping: update to 1.17.3.

### PR DESCRIPTION
**Pull request by icp1994**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture: **x86_64**

### CODE REVIEW

**Comment by tranzystorekk** [comment]
We could alternatively patch Cargo.toml to disable git2 from the shadow-rs crate, especially since there's no git repo to inspect after all.

**Comment by icp1994** [comment]
Those patches get annoying to carry over versions. I sent a patch upstream for now.

File: srcpkgs/gping/template [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Template file for 'gping'
 pkgname=gping
 version=1.16.1
-revision=1
+revision=2
 build_style=cargo
 make_install_args="--path gping"
+hostmakedepends="pkg-config libgit2-devel"
+makedepends="libgit2-devel"
```
Comment (tranzystorekk) [inline srcpkgs/gping/template:8]: if it's just the build script then we need only hostmakedepends

Comment (icp1994) [reply srcpkgs/gping/template:8]: Tried that, [didn't work](https://github.com/void-linux/void-packages/actions/runs/8534367101/job/23378632021) for cross. It's not only for build, also in [main]( https://github.com/orf/gping/blob/gping-v1.6.1/gping/src/main.rs#L36).

**Comment by Piraty** [comment]
ping

**Pull request merged**